Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -212,25 +201,28 @@ public void periodic( )
 
     if (m_useLimelight)
     {
+      //DataLogManager.log(String.format("using limelight"));
```

Comment:
This datalog print message will never be a good idea due to it spamming the console. It was useful when debugging once, but probably want to clean this out.